ui/apps/platform/src/Containers/NetworkGraph/deployment/ExternalFlows.tsx

```diff
@@ -1,65 +1,101 @@
-import React, { useCallback } from 'react';
+import React from 'react';
 
-import { UseURLPaginationResult } from 'hooks/useURLPagination';
-import useRestQuery from 'hooks/useRestQuery';
-import { UseUrlSearchReturn } from 'hooks/useURLSearch';
-import { getExternalIpsFlowsMetadata } from 'services/NetworkService';
-import { ExternalNetworkFlowsMetadataResponse } from 'types/networkFlow.proto';
-import { getTableUIState } from 'utils/getTableUIState';
+import {
+    ExpandableSection,
+    ExpandableSectionToggle,
+    Stack,
+    StackItem,
+    Toolbar,
+    ToolbarContent,
+    ToolbarItem,
+} from '@patternfly/react-core';
+import pluralize from 'pluralize';
 
-import ExternalIpsTable from '../external/ExternalIpsTable';
-import { NetworkScopeHierarchy } from '../types/networkScopeHierarchy';
+import { TimeWindow } from 'constants/timeWindows';
+import useSelectToggle from 'hooks/patternfly/useSelectToggle';
+
+import FlowsTableHeaderText from '../common/FlowsTableHeaderText';
+import { FlowTable } from '../components/FlowTable';
+import { useNetworkBaselineStatus } from '../hooks/useNetworkBaselineStatus';
 
 type ExternalFlowsProps = {
-    deploymentName: string;
-    scopeHierarchy: NetworkScopeHierarchy;
-    urlSearchFiltering: UseUrlSearchReturn;
-    urlPagination: UseURLPaginationResult;
-    onExternalIPSelect: (externalIP: string) => void;
+    deploymentId: string;
+    timeWindow: TimeWindow;
 };
 
-function ExternalFlows({
-    deploymentName,
-    scopeHierarchy,
-    urlSearchFiltering,
-    urlPagination,
-    onExternalIPSelect,
-}: ExternalFlowsProps) {
-    const clusterId = scopeHierarchy.cluster.id;
-    const { namespaces } = scopeHierarchy;
-    const { searchFilter } = urlSearchFiltering;
-    const { page, perPage } = urlPagination;
-    const fetchExternalIpsFlowsMetadata =
-        useCallback((): Promise<ExternalNetworkFlowsMetadataResponse> => {
-            return getExternalIpsFlowsMetadata(clusterId, namespaces, [deploymentName], {
-                sortOption: {},
-                page,
-                perPage,
-                advancedFilters: searchFilter,
-            });
-        }, [page, perPage, clusterId, deploymentName, namespaces, searchFilter]);
+function ExternalFlows({ deploymentId, timeWindow }: ExternalFlowsProps) {
+    const anomalous = useNetworkBaselineStatus(deploymentId, timeWindow, 'ANOMALOUS');
+    const baseline = useNetworkBaselineStatus(deploymentId, timeWindow, 'BASELINE');
```

Comment:
An observation, not a change request. IF we had confidence earlier that the information and interaction design was stable, it might have been worth considering pro and con to have separate child components like `ExternalFlowsAnomalous` and `ExternalFlowsBaseline` compared to the family of twins at this level.

Although that might have made a bit extra effort to lift up error message in one of the companion PRs.